microsoft-edge/index.yml

```diff
@@ -33,7 +33,7 @@ highlightedContent:
     # Card
     - title: "Find out what's new"
       itemType: learn
-      url: https://www.microsoftedgeinsider.com/whats-new
+      url: https://www.microsoft.com/edge/download/insider
```

Comment:
orig url:
https://www.microsoftedgeinsider.com/whats-new
goes to:
https://www.microsoft.com/en-us/edge/download/insider?form=MA13FJ

Reply:
fixed, added 8th card